#llm_api_type: google
llm_api_type: openai
llm_api_base_url: https://api.groq.com/openai/v1
#vlm_model: gemini-1.5-flash
vlm_model: llama3-8b-8192
#manipulator_model: doris_arm
#manipulator_model: wx200
prompt_repo: crislmfroes/fbot-vlm-agent
max_steps: 25